tuf/repository_lib.py

```diff
@@ -1869,8 +1905,11 @@ def sign_metadata(metadata_object, keyids, filename):
     if key['keytype'] in SUPPORTED_KEY_TYPES:
       if 'private' in key['keyval']:
         signed = signable['signed']
-        signature = tuf.keys.create_signature(key, signed)
-        signable['signatures'].append(signature)
+        try:
+          signature = tuf.keys.create_signature(key, signed)
+          signable['signatures'].append(signature)
+        except:
```

Comment:
We should catch specific exception(s) here, or at least log the caught exception if the bare `except` is used.